resources/release-drafter.md

```diff
@@ -20,22 +20,19 @@ The draft release description is therefore generated and corresponds to all the
 
 MeiliSearch tools follow the [Semantic Versioning Convention](https://semver.org/). Based on the chosen labels for a PR, the release-drafter automatically increases the right number in the version.
 
-### Skip the PR
+### Label the PR
 
-If you don't want a PR to appear in the release changelogs: add the label `skip-changelog`.
-
-We suggest removing PRs updating the README or the CI. Users don't need this kind of information when updating the package.
+Each PR should be labelled by one of the following labels: `skip-changelog`, `breaking-change`, `security`, `enhancement` or `bug`.
 
-### The PR adds a new feature
+According to the label you choose, the changelog will be put in the related section in the release draft. It means you cannot mix them in the same PR.
```

Comment:
Same question, if I want to put two labels what happens?

Reply:
> What do you mean? If I add security and enhancement il would only appear in one of them?

I thought it would choose one of both sections. But I might had bad tested, because it puts the PRs in both sections. Is this a behavior ok for you @bidoubiwa @alallema?
Should we remove the last sentence then?

Reply:
Maybe just said that if you mixed multiple labels it would appear in multiple sections?